sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_example.ipynb

```diff
@@ -38,22 +38,22 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        KeyName="ec2-key-pair",
</pre><p>It would be better to add a timestamp to this key name. If the customer already has this generic key name, it will get deleted later which could be catastrophic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I didn't want to alter behavior too much, but this seems very important.